### ISSUE TITLE
Changing the viewport with buttons

### ISSUE THREAD

**Issue by nirchernia** [namespace-ee/react-calendar-timeline#38]
Here is a simple example:

```
import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import './index.css';
import Timeline from 'react-calendar-timeline'
import moment from 'moment'

const groups = [
  {id: 1, title: 'group 1'},
  {id: 2, title: 'group 2'}
]

const items = [
  {id: 1, group: 1, title: 'item 1', start_time: moment(), end_time: moment().add(1, 'hour')},
  {id: 2, group: 2, title: 'item 2', start_time: moment().add(-0.5, 'hour'), end_time: moment().add(0.5, 'hour')},
  {id: 3, group: 1, title: 'item 3', start_time: moment().add(2, 'hour'), end_time: moment().add(3, 'hour')}
]

var start = moment().add(-12, 'hour')
var end = moment().add(12, 'hour')

function handleClick(){
    console.log('you clicked')
    start = moment(start).add(12, 'hour')
    end = moment(end).add(12,'hour')
    handleTimeChange(start, end)
}

function handleTimeChange(visibleTimeStart, visibleTimeEnd){
    console.log(visibleTimeStart, visibleTimeEnd)
    start = visibleTimeStart
    end = visibleTimeEnd
    this.updateScrollCanvas(start, end)

}

ReactDOM.render(<div>
  <a href="#" onClick={handleClick}>Move Forward</a>
  <div>
    Rendered by react!
    <Timeline groups={groups}
              items={items}
              defaultTimeStart={start}
              defaultTimeEnd={end}
              visibleTimeStart={start.unix()}
              visibleTimeEnd={end.unix()}
              onTimeChange={handleTimeChange}


              />
  </div></div>,
  document.getElementById('root')
);
```

When clicking on `<a href="#" onClick={handleClick}>Move Forward</a>`, I receive the error: `Uncaught TypeError: Cannot read property 'updateScrollCanvas' of undefined`. That makes sense. Since there is no **this** in the example case. But how can I control/update the viewport with functions outside of `Timeline`? 

**tl;dr:** I want my button `Move Forward` to be able to move the viewport start and end 12 hours forward, what is the best way to do this?

**Comment by nandito** [comment]
I'm stucked with almost the same problem, the onTimeChange cannot access the updateScrollCanvas.
I did some debugging:
modules/lib/Timeline.js:1029
```
  onTimeChange: function onTimeChange(visibleTimeStart, visibleTimeEnd) {
    this.updateScrollCanvas(visibleTimeStart, visibleTimeEnd);
},
```
If I add a `console.log(this)` inside this function, I see that I the result object does not contain updateScrollCanvas
How can I update the scrollCanvas?

**Comment by mariusandra** [comment]
Hey, I have added `updateScrollCanvas` as the third parameter of `onTimeChange`, so you can call it like that. It's out with v0.9.0
